### PR TITLE
Fix padding issues in views (i.e. make view - settings)

### PR DESCRIPTION
**Pull request by droxxr**
- Some minor markup and css changes
- Fix padding issues in make view (settings)

Important: I think we should use the main html tag to contain the components in our views, so that i'll be easier to apply paddings

### CODE REVIEW

File: static/styles/forms.less [hunk 1]
```diff
@@ -108,8 +108,7 @@ p.url {
 
 .form-group,
 .checkbox {
-    // Left and right padding should be set in the view
-    padding: @body-padding 0px;
+    padding: @body-padding 0; // The left and right padding is defined by the main-container element of a view
```
Comment (k88hudson) [inline static/styles/forms.less]: :+1:

File: views/make/index.less [hunk 2]
```diff
@@ -1,10 +1,14 @@
 #make {
     margin-bottom: @editBar-height;
 
-    .edit-inner {
+    .edit-container {
         padding: 10px;
     }
 
+    .settings-container {
+        padding: 0px 15px;
```
Comment (k88hudson) [inline views/make/index.less]: no `px` needed

**Comment by k88hudson** [comment]
@droxxr i noticed the padding on the share view is messed up, can you fix that too?

This is great, thank you

**Comment by droxxr** [comment]
@k88hudson I made the changes, and added padding to the share view.

**Comment by k88hudson** [comment]
needed a rebase for something we just merged, so i merged here https://github.com/mozilla/webmaker-app/pull/701

Thanks! :dancer:

**Pull request closed without merging**